frontend/src/components/common/Game/Dialog/EditRom.vue

```diff
@@ -225,6 +225,14 @@ function closeDialog() {
     <template #append>
       <v-row class="justify-center mt-4 mb-2" no-gutters>
         <v-btn-group divided density="compact">
+          <v-btn
+            v-if="rom.igdb_id || rom.moby_id || rom.sgdb_id"
```

Comment:
in most cases we want buttons visible but disabled

```suggestion
           :disabled="rom.igdb_id || rom.moby_id || rom.sgdb_id"
```